src/server/cph-number/controller.js

```diff
@@ -28,7 +28,10 @@ export const postController = {
   handler(req, res) {
     const valid = /([0-9]{2})\/([0-9]{3})\/([0-9]{4})/g
     let errorMessage = null
-    const { cphNumber } = /** @type {CphNumberPayload} */ (req.payload)
+    let { cphNumber } = /** @type {CphNumberPayload} */ (req.payload)
+
+    // Remove whitespace from cphNumber
+    cphNumber = cphNumber ? cphNumber.replace(/\s+/g, '') : ''
```

Comment:
happy to pull it through as one this time, but would prefer a separate PR for this typically.

Reply:
@eoin-corr-git  - are we looking to add a test for this case in the user journey tests?

Reply:
totally agree!!